test/pkg.jl

```diff
@@ -100,6 +100,8 @@ temp_pkg_dir() do
     str = chomp(takebuf_string(iob))
     @test endswith(str, string(Pkg.installed("Example")))
     @test isempty(Pkg.dependents("Example"))
+    # Checking to see dependency reading
+    @test Pkg.dependencies("Example") == Dict("julia" => Pkg.Types.VersionSet(v"0.3.0", typemax(VersionNumber)))
```

Comment:
What about cases where you are running an older version or on a branch? I'm not sure if returning information about anything other than what is installed is more predictable. Unless you mean something like "returns the same thing on all systems" by predictable?

I do agree it would make sense for the user to be able to specify whether you want dependencies for the latest or installed version. Maybe something like:

```julia
function dependencies(packagename::AbstractString, version::Union{VersionNumber, Symbol}=:installed)
    cd(Pkg.dir()) do 
        if version==:installed
            reqpath = joinpath(Pkg.dir(packagename), "REQUIRE")
            if !isfile(reqpath)
                throw(PkgError("No file found for installed package $packagename at $reqpath."))
            end
            Pkg.Reqs.parse(readlines(reqpath))
        elseif version==:latest
            Pkg.Read.latest()[packagename].requires
        elseif isa(version, VersionNumber)
            Pkg.Read.available(packagename)[version].requires
        else
            throw(SomeError("Couldn't find $packagename $version"))
        end
    end    
end
```